fusor-ember-cli/app/models/deployment-plan.js

```diff
@@ -3,133 +3,22 @@ import DS from 'ember-data';
 
 export default DS.Model.extend({
   name: DS.attr('string'),
-  uuid: DS.attr('string'),
-  roles: DS.hasMany('deployment-role', {inverse: null, async: false}),
-  parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false}),
+  parameters: DS.attr(),
+  roles: DS.attr(),
```

Comment:
@cfchase, what's the advantage of using `DS.attr()` with no data type verses `DS.hasMany`?